.github/workflows/testing.yml

```diff
@@ -98,7 +95,9 @@ jobs:
       # Only run vet for 'vet' runs.
       - name: Run vet.sh
         if: matrix.type == 'vet'
-        run: ./vet.sh -install && ./vet.sh
+        run: |
+          VET_SKIP_PROTO=1
+          ./vet.sh -install && ./vet.sh
```

Comment:
Nit: I think these can fit on one line.

Reply:
I dont know of a clean one line way to apply `VET_SKIP_PROTO` to both `./vet.sh` commands

Reply:
oh ... you need it set for both? :(

If you're interested, I've been meaning to delete `VET_SKIP_PROTO` and instead make it the default, with `VET_CHECK_PROTO=1` set if you actually what to check the proto files.

Reply:
That's a good idea. I can send a follow for that.

Reply:
https://github.com/grpc/grpc-go/pull/7099

-- I'll actually wait for that to merge before merging this PR